ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/DOWNLINK_IPV4_POOL_TESTS.yml

```diff
@@ -0,0 +1,53 @@
+default_node_types:
+  - node_type: spine
+    match_hostnames:
+      - .*spine\d
+  - node_type: l3leaf
+    match_hostnames:
+      - .*l3leaf\d
+
+spine:
+  defaults:
+    loopback_ipv4_pool: 192.168.0.0/24
+    bgp_as: 65000
```

Comment:
Do we support having `downlink_pools` here? Per the schema it says so but when testing locally I am getting an error:
```
spine:
  defaults:
    platform: cEOSLab
    loopback_ipv4_pool: 10.1.0.0/24
    bgp_as: 65100
    downlink_pools:
      - downlink_ipv4_pool: 172.21.0.0/16
        downlink_interfaces: [Ethernet1-4]
 ```
 Duplicate objects error:
 ```bash
 An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ansible_collections.arista.avd.plugins.plugin_utils.errors.errors.AristaAvdDuplicateDataError: Found duplicate objects with conflicting data while generating configuration for IP address defined under BGP neighbor for underlay. {'ip_address': '172.21.0.0', 'peer_group': 'IPv4-UNDERLAY-PEERS', 'description': 'DC1-SPINE2_Ethernet1', 'peer': 'DC1-SPINE2'} conflicts with {'ip_address': '172.21.0.0', 'peer_group': 'IPv4-UNDERLAY-PEERS', 'description': 'DC1-SPINE1_Ethernet1', 'peer': 'DC1-SPINE1'}.
fatal: [DC1-LEAF1A -> localhost]: FAILED! => changed=false 
  msg: 'Found duplicate objects with conflicting data while generating configuration for IP address defined under BGP neighbor for underlay. {''ip_address'': ''172.21.0.0'', ''peer_group'': ''IPv4-UNDERLAY-PEERS'', ''description'': ''DC1-SPINE2_Ethernet1'', ''peer'': ''DC1-SPINE2''} conflicts with {''ip_address'': ''172.21.0.0'', ''peer_group'': ''IPv4-UNDERLAY-PEERS'', ''description'': ''DC1-SPINE1_Ethernet1'', ''peer'': ''DC1-SPINE1''}.'
```

Reply:
Same error if we put them under `node_groups`:
```
  node_groups:
    - group: SPINES
      downlink_pools:
        - downlink_ipv4_pool: 172.21.1.0/24
          downlink_interfaces: [Ethernet1-2]
        - downlink_ipv4_pool: 172.21.2.0/24
          downlink_interfaces: [Ethernet3-4]
      nodes:
        - name: DC1-SPINE1
          id: 1
          mgmt_ip: 172.20.20.101/24
        - name: DC1-SPINE2
          id: 2
          mgmt_ip: 172.20.20.102/24
```

Reply:
Yeah we need to move it out of the common variables and into the specific section for `nodes` since it will only work per device.

Reply:
Is this issue fine then? Or is there something we need to do about this.